embulk-core/src/main/java/org/embulk/spi/Row.java

```diff
@@ -0,0 +1,318 @@
+package org.embulk.spi;
+
+import org.embulk.spi.time.Timestamp;
+import org.embulk.spi.type.Type;
+import org.embulk.spi.type.Types;
+import org.msgpack.value.Value;
+
+/**
+ * This class is a temporal container for storing a row only.
```

Comment:
IIUC reading the code, the purpose of this class `Row` is not to double-store values for the same columns, right?

If so, I couldn't get the point from this comment and the PR description. What do you think to comment:
```
/**
 * Row is a container to stage values before adding into reference lists such as |stringReferences|.
 *
 * |Row| works as a buffer against plugins that may add values incorrectly without |PageBuilder#addRecord|.
 * It accepts just one value per column while |PageBuilder| can double-store values regardless of columns.
 * Double-stored values are overwritten.
 */
```